Makefile

```diff
@@ -68,15 +68,14 @@ lint: ## Run golangci-lint against code.
 
 .PHONY: test-unit
 test-unit: manifests generate fmt vet envtest ## Run unit tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... -tags=unit -coverprofile cover-unit.out -v
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(shell find ./pkg/_internal -mindepth 1  -type d) ./...  -tags=unit -coverprofile cover-unit.out
```

Comment:
I see how this solves the problem of running the tests.
Why not just rename the folder?
Is there something special about the name _internal that it has to be kept that way?

Reply:
Agree, we need to see if we can just rename that directory. I'm not sure of the reason behind using the underscore there. @maleck13 might know better. Other repos that i can see have `internal` package directories, but they don't contain the underscore see [cert-manager](https://github.com/cert-manager/cert-manager/tree/master/internal ) for example.

Probably a follow up issue/pr to resolve that particular issue would be better.

Reply:
we could do either way. I, personally, don't have a preference there.

Reply:
That said, if this is a temporary workaround and it's going to make the tests run on CI, I'm OK with just doing it this way for now, although a bit odd.